tests/services/test_schemas_json_load.py

```diff
@@ -7,18 +7,16 @@
 # it under the terms of the MIT License; see LICENSE file for more details.
 
 """DEPRECATED. Tests for Invenio RDM Records JSON Schemas."""
-import os
```

Comment:
This file can be deleted when all the schemas have their own test file like above. A gradual approach might work well:
- remove ResourceTypeSchema from this file because covered above and so on...